code/game/jobs/access.dm

```diff
@@ -7,14 +7,23 @@
 		return FALSE
 	return check_access_list(M.GetAccess())
 
-/atom/movable/proc/GetAccess()
+/atom/movable/proc/GetAccess(var/union = FALSE)
```

Comment:
What is the purpose of changing the default behavior of GetAccess() to not be a union of all available accesses?

Reply:
EDIT: It *was* originally a hardcoded union but only on humantypes. Now, humantypes use the argument and have it default to true.